### PR TITLE
Updated test dependencies

### LINKED ISSUE #181: chore: Update test dependencies
Python environment is being updated to 3.11, [issue](https://github.com/kytos-ng/kytos/issues/448). Taking the oportunity to update this NApp dependencies too.

### PR DESCRIPTION
**Pull request by Alopalao**
Closes #181 

### Summary

Updated test dependencies
Depends in kytos [PR](https://github.com/kytos-ng/kytos/pull/460)

### Local Tests
Run tox and kytos

### End-to-End Tests
TBA

### CODE REVIEW

File: db/models/__init__.py [hunk 1]
```diff
@@ -40,54 +41,55 @@ def dict(self, **kwargs) -> dict:
 class FlowCheckDoc(DocumentBaseModel):
     """FlowCheckDoc."""
 
-    state = "active"
+    state: str = Field(default="active")
```
Comment (viniarck) [inline db/models/__init__.py:44]: Looks like this can be just `state: str = "active"`, doesn't it?

File: db/models/__init__.py [hunk 2]
```diff
@@ -105,37 +107,34 @@ def vlan_with_mask(cls, v):
...
-    actions: Optional[List[dict]]
-    instructions: Optional[List[dict]]
-
-    class Config:
-        """Config."""
-
-        arbitrary_types_allowed = True
-
-    @validator("cookie", pre=True)
+    table_id: int = Field(default=0)
```
Comment (viniarck) [inline db/models/__init__.py:110]: Same comments regarding the `Field` usages below. Looks like most of them can be assigned directly without `Field(default)`. Can you verify this again?

File: db/models/__init__.py (hunk 1 above)
Comment (Alopalao) [reply db/models/__init__.py:44]: Yes, changed

File: db/models/__init__.py (hunk 2 above)
Comment (Alopalao) [reply db/models/__init__.py]: Changed this too.

**Review by viniarck — APPROVED** [review]
LGTM. Leaving it pre-approved to land soon.

**Pull request merged**